[2024-10-20T02:49:13.328+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl.load manual__2024-10-20T02:49:07.667430+00:00 [queued]>
[2024-10-20T02:49:13.341+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl.load manual__2024-10-20T02:49:07.667430+00:00 [queued]>
[2024-10-20T02:49:13.341+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-10-20T02:49:13.357+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load> on 2024-10-20 02:49:07.667430+00:00
[2024-10-20T02:49:13.364+0000] {standard_task_runner.py:57} INFO - Started process 145 to run task
[2024-10-20T02:49:13.367+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl', 'load', 'manual__2024-10-20T02:49:07.667430+00:00', '--job-id', '191', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmpfjdei3wc']
[2024-10-20T02:49:13.370+0000] {standard_task_runner.py:85} INFO - Job 191: Subtask load
[2024-10-20T02:49:13.431+0000] {task_command.py:416} INFO - Running <TaskInstance: crypto_etl.load manual__2024-10-20T02:49:07.667430+00:00 [running]> on host 6fb9dd1ce83c
[2024-10-20T02:49:13.520+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-10-20T02:49:07.667430+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-20T02:49:07.667430+00:00'
[2024-10-20T02:49:13.578+0000] {logging_mixin.py:154} INFO - Success: Loaded 101 recall records to recalls_db.
[2024-10-20T02:49:13.579+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-10-20T02:49:13.591+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=crypto_etl, task_id=load, execution_date=20241020T024907, start_date=20241020T024913, end_date=20241020T024913
[2024-10-20T02:49:13.623+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-10-20T02:49:13.643+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
